### PR TITLE
DOC: GNU-Make on Windows

### PR DESCRIPTION
**Pull request by marsipu**
#### Reference issue
Extends #8027.

#### What does this implement/fix?
- Added GNU-Make-Installation for Windows
- Git-Installation for Windows more clear
- Fixed small typo in "running the tests"

### CODE REVIEW

File: doc/install/contributing.rst [hunk 1]
```diff
@@ -121,6 +121,46 @@ There are many other ways to customize git's behavior; see `configuring git`_
 for more information.
 
 
+GNU Make
+~~~~~~~~
+GNU Make facilitates deploying a package by executing corresponding commands
+from the ``Makefile``. For MNE-Python we got two Makefiles, one in the parent
```
Comment (agramfort) [inline doc/install/contributing.rst:127]:
```suggestion
from the ``Makefile``. For MNE-Python we have two Makefiles, one in the parent
```

**Review by agramfort — APPROVED** [review]

**Comment by agramfort** [comment]
@larsoner or @drammock I let you merge if you feel like

File: doc/install/contributing.rst [hunk 2]
```diff
@@ -121,6 +121,46 @@ There are many other ways to customize git's behavior; see `configuring git`_
...
+
+To check if make is already installed type ::
+
+   $ make
+
+into a terminal and you should see ::
+
+   make: *** No targets specified and no makefile found.  Stop.
+
+If you don’t see this or something similar:
```
Comment (cbrnr) [inline doc/install/contributing.rst:138]:
```suggestion
If you don't see this or something similar:
```

File: doc/install/contributing.rst [hunk 3]
```diff
@@ -121,6 +121,45 @@ There are many other ways to customize git's behavior; see `configuring git`_
...
+
+   - ``(Anaconda-Path)``
+   - ``(Anaconda-Path)\Scripts``
+
+   to Windows-PATH.
+
+- For Linux/MacOS, get `GNU Make`_
+- For Windows, you can install make for git BASH (which comes with `git for Windows`_):
+
+   1. Download ``make-(newest version)-without-guile-w32-bin.zip`` from `ezwinports`_
```
Comment (larsoner) [inline doc/install/contributing.rst:155]: There is one too many spaces on these lines that make it render as a blockquote:

https://21851-1301584-gh.circle-artifacts.com/0/dev/install/contributing.html#gnu-make

Comment (marsipu) [reply doc/install/contributing.rst:155]: Thank you, I am fixing it

**Comment by larsoner** [comment]
Thanks @marsipu !

**Pull request merged**